hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableTokenRelationStoreImpl.java

```diff
@@ -49,9 +51,11 @@ public ReadableTokenRelationStoreImpl(@NonNull final ReadableStates states) {
      * {@inheritDoc}
      */
     @Override
-    public Optional<TokenRelation> get(final long accountNum, final long tokenNum) {
-        final var tokenRelation =
-                Objects.requireNonNull(readableTokenRelState).get(EntityNumPair.fromLongs(accountNum, tokenNum));
+    public Optional<TokenRelation> get(@NonNull final AccountID accountId, @NonNull final TokenID tokenId) {
+        if (AccountID.DEFAULT.equals(accountId) || TokenID.DEFAULT.equals(tokenId)) return Optional.empty();
+
+        final var tokenRelation = Objects.requireNonNull(readableTokenRelState)
+                .get(EntityNumPair.fromLongs(accountId.accountNum(), tokenId.tokenNum()));
```

Comment:
Now that it's a thing, might as well `requireNonNull(accountId).accountNum()` etc here.

Reply:
We can. It doesn't fix the compiler warning though, so what would be the use for this?

Reply:
Done